lib/attr_encrypted.rb

```diff
@@ -302,7 +302,7 @@ def encrypted_attributes
   #
   #   User.encrypt_email('SOME_ENCRYPTED_EMAIL_STRING')
   def method_missing(method, *arguments, &block)
-    if method.to_s =~ /^((en|de)crypt)_(.+)$/ && attr_encrypted?($3)
+    if method.to_s =~ /^(legacy_(en|de)crypt)_(.+)$/ && attr_encrypted?($3)
```

Comment:
This could be a problem to change this if we used these methods, which we don't at the moment.
I changed this because ActiveRecord provides now the same naming.